infrastructure/main.tf

```diff
@@ -37,14 +37,14 @@ resource "vault_generic_secret" "servicebus-listen-conn-string" {
 
 module "db" {
   source              = "git@github.com:contino/moj-module-postgres.git?ref=feature/specify-db-name"
```

Comment:
The branch must be master and postgresql_database must be `postgres`, unfortunately